basic/http/collect-user.test.js

```diff
@@ -1,13 +1,25 @@
 const request = require('supertest');
-const app = require('./collect-user');
+const { app, hashPassword,argon2 } = require('./collect-user');
 
 describe('app', () => {
+  afterAll(() => {
+    server.close();
+  });
+  
+
   const server = app.listen(8080);
+
   
   it('should have the response status of 200', async () => {
     const response = await request(server).get('/users');
     expect(response.status).toBe(200)
   });
+  it('should accept string password', async () => {
+    const str = 'testing';
+    const hashedPassword = hashPassword(str);
+    const verified = await argon2.verify("<big long hash>", "password");
```

Comment:
the `<big long hash>` should be the `hashedPassword` in order to check if we're using the hashing algorithm of `argon`.

```ts
const verified = await argon2.verify(hashedPassword str);
```

Reply:
RESOLVED 💯